.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - 'stable'
+cache:
+  directories:
+    - node_modules
+script:
+  - npm test
```

Comment:
Please add a new line to this file

Reply:
> I think you forgot to add new lines to the end of files, kindly add them and it would be fine.

eslint as not been setup for this project, so I couldn't spot that

Reply:
That's fine but I don't think eslint outrightly affects adding new lines on a file.